recipes/r-bdrc/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-bdrc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bdrc_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bdrc/bdrc_{{ version }}.tar.gz
+  sha256: 2cfb02988afa0a9b9e1cd73c610e81800a5891d56e21deda7479338409a87876
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-ggplot2
+    - r-gridextra
+    - r-rlang
+    - r-scales
+  run:
+    - r-base
+    - r-ggplot2
+    - r-gridextra
+    - r-rlang
+    - r-scales
+
+test:
+  commands:
+    - $R -e "library('bdrc')"           # [not win]
+    - "\"%R%\" -e \"library('bdrc')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=bdrc
+  license: MIT
+  summary: Fits a discharge rating curve based on the power-law and the generalized power-law
+    from data on paired stage and discharge measurements in a given river using a Bayesian
+    hierarchical model as described in Hrafnkelsson et al. (2020) <arXiv:2010.04769>.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```

`DESCRIPTION` shows `License: MIT + file LICENSE`, so should also capture the LICENSE file:

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
    - LICENSE
```

(Note that it is in the tarball and will be captured from there automatically.)